index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width" />
+    <meta
+      name="description"
+      content="Author: Amedzro Emmanuel Elikplim, School: Microverse, Project: Portfolio"
+    />
+
+    <link rel="stylesheet" href="styles.css" />
+
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      rel="preload"
+      as="style"
+      onload="this.onload = null; this.rel = 'stylesheet'"
+      href="https://fonts.googleapis.com/css2?family=Raleway:wght@100;500&family=Roboto&display=swap"
+    />
+
+    <link
+      rel="preload"
+      as="style"
+      onload="this.onload = null; this.rel = 'stylesheet'"
+      href="https://use.fontawesome.com/releases/v5.15.4/css/all.css"
+      integrity="sha384-DyZ88mC6Up2uqS4h/KRgHuoeGwBcD4Ng9SiP4dIRy0EXTlnuz47vAwmeGwVChigm"
+      crossorigin="anonymous"
+    />
+
+    <title>Portfolio</title>
+  </head>
+
+  <body>
+    <nav class="toolbar">
+      <ul class="toolbar-items">
+        <li><h3 id="logo-text">John Doe</h3></li>
+        <li><i class="fa fa fa-bars bar-icon"></i></li>
+      </ul>
+    </nav>
```

Comment:
- [ ] To keep your HTML semantic, you should consider the following structure:
```
<header class="toolbar">
  <h3 id="logo-text">John Doe</h3>
  <nav>
    <i class="fa fa fa-bars bar-icon"></i>
   </nav>
</header>
```
- [ ] You should consider having the whole section wrapped in a `<header>` tag. Even though you don't have a functional menu button yet, it will hold all your navigation links, so it makes more sense to have that wrapped in the `<nav>` tag.

> In the following links you can read more about [header](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header) and [nav](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nav) tags.

Reply:
`<header class="toolbar">
  <h3 id="logo-text">John Doe</h3>
  <nav>
    <i class="fa fa fa-bars bar-icon"></i>
   </nav>
</header>`

With the code above, the h3 tag is outside of the nav tag. I updated the code like this:

`<header>
      <nav class="toolbar">
        <ul class="toolbar-items">
          <li><h3 id="logo-text">John Doe</h3></li>
          <li><i class="fa fa fa-bars bar-icon"></i></li>
        </ul>
      </nav>
 </header>`

This wraps both h3 and 'i' tag in the nav tag. I preferred using ul and li tags in navbars since it's easier to add more nav links in the desktop mode.
Thanks

Reply:
```
<header class="toolbar">
  <h3 id="logo-text">John Doe</h3>
  <nav>
    <i class="fa fa fa-bars bar-icon"></i>
   </nav>
</header>
``` 
In the code above the h3 element is outside of the nav tag this changes the position of `John Doe` in the header tag.
I updated my code as follow:

```
    <header>
      <nav class="toolbar">
        <ul class="toolbar-items">
          <li><h3 id="logo-text">John Doe</h3></li>
          <li><i class="fa fa fa-bars bar-icon"></i></li>
        </ul>
      </nav>
    </header>
```
The `nav` tag is wrapped by the `header` tag. Furthermore, I prefer using `ul` and `li` in the navbar, since it makes it easier to add additional nav links if neccessary